### PR TITLE
[CWS] fix secl weight

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

Fix an operator weight in SECL.

### CODE REVIEW

File: pkg/security/secl/compiler/eval/operators.go [hunk 1]
```diff
@@ -352,7 +352,7 @@ func StringEquals(a *StringEvaluator, b *StringEvaluator, state *State) (*BoolEv
 
 		return &BoolEvaluator{
 			Value:           op(ea, eb),
-			Weight:          a.Weight + InArrayWeight*len(eb),
+			Weight:          a.Weight,
```
Comment (paulcacheux) [inline pkg/security/secl/compiler/eval/operators.go:355]: Why is this only related to a and not b ?

**Review by spikat — APPROVED** [review]
Could it be nice to have some UT for this ?

File: pkg/security/secl/compiler/eval/operators.go (hunk 1 above)
Comment (safchain) [reply pkg/security/secl/compiler/eval/operators.go]: Removed a & b as they don't depend on a handler

**Comment by safchain** [comment]
@spikat I added a UT

**Review by paulcacheux — APPROVED** [review]
